src/__tests__/third-party/baseweb/pages/_app.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import App from 'next/app';
+import { Provider as StyletronProvider } from 'styletron-react';
+import { LightTheme, BaseProvider } from 'baseui';
+import { styletron } from '../styletron';
+
+export default class MyApp extends App {
+  render() {
+    const { Component, pageProps } = this.props;
+    return (
+      <StyletronProvider value={styletron}>
```

Comment:
The problem seems to happen when wrapped with `<StyletronProvider value={styletron}>`